implementations.yaml

```diff
@@ -6,7 +6,7 @@ traits_version: 0.3.2
 kems:
   kyber:
     version: 0.6.5
-    avx2_guard: 'cfg(all(not(disable_avx2), not(target_os = "windows"), not(target_os = "macos"), target_arch = "x86_64"))'
+    avx2_guard: '!is_windows && !is_macos && target_arch == "x86_64"'
```

Comment:
This is also not good enough: if you have an old CPU this will lead to illegal instructions.

Reply:
Actually I should first have coffee before deciding such things — the above isn't true.